lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -0,0 +1,105 @@
+from flask_restful import Resource, abort
+from flask import jsonify
+
+from pbench.server.api.resources.query_apis import get_index_prefix
+
+
+class EndpointConfig(Resource):
+    """
+    EndpointConfig API resource: this supports dynamic dashboard configuration
+    from the Pbench server rather than constructing a disconnected dashboard
+    config file.
+    """
+
+    def __init__(self, config, logger):
+        """
+        __init__ Construct the API resource
+
+        Args:
+            config (PbenchServerConfig): server config values
+            logger (Logger): message logging
+        """
+        self.logger = logger
+        self.host = config.get("pbench-server", "proxy_host")
+        uri_prefix = config.rest_uri
+        self.uri = f"{self.host}{uri_prefix}"
+        self.prefix = get_index_prefix(config)
+        self.commit_id = config.COMMIT_ID
+
+    def get(self):
+        """
+        Return server configuration information required by the UI
+        dashboard. This includes
+
+        metadata: Information about the server configuration
+            prefix: The indexing prefix applied by the server to partition a
+                shared Elasticsearch cluster index namespace.
+            result_index: The "root" index name for Pbench result data,
+                qualified by the current index version and prefix. In the
+                current ES schema, this is "v5.result-data-sample."
```

Comment:
You put periods at the end of the description at lines 35-36 and elsewhere below, but for descriptions ending in `."` you omitted the trailing periods.  This is conventional typography for prose, but it creates an ambiguity in this context.  Thus, I would recommend either omitting the periods from all the descriptions (many of them are not full sentences, after all) or supplying periods on all of them (which will make it clear that the text inside the quotes is literal).